### DISCUSSION TITLE
Make DNSSEC enabled on all alias domain and reachable via ipv6

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#888] [Feature Requests]
After a quick test i have found out some alias domain do not have dnssec enabled domain, and the email server isn't reachable via ipv6.
I thinks these two "feature" must be there in 2022.

What's your though

Note: i think dnssec is a config mistake, but ipv6 not.

**Comment by nguyenkims** [comment]
@BirdInFire ipv6 isn't well supported in the email world unfortunately and that's why we decided to only support ipv4. When email softwares have a better support for ipv6, we'll definitely turn on its support.

About the DNSSEC, we'll check with our infra team to enable them, thanks for the info!
